Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,329 @@
+
+..  confval:: cache
+    :name: pageview-cache
+    :type: :ref:`cache <cache>`
+
+    See :ref:`cache function description <cache>` for details.
+
+..  _cobj-pageview-dataProcessing:
+
+dataProcessing
+--------------
+
+..  confval:: dataProcessing.[key]
+    :name: pageview-dataProcessing
+    :type: :ref:`data-type-path` with :ref:`stdWrap <stdwrap>`
+
+    Add one or multiple :ref:`data processors <dataProcessing>`. The
+    sub-property :typoscript:`options` can be used to pass parameters to the
+    processor class.
+
+Example: Display a main menu and a breadcrumb on the page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  literalinclude:: _includes/_pageWithBreadcrumb.typoscript
+    :language: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+
+The page template could look like this:
+
+..  literalinclude:: _includes/_PageWithBreadcrumb.html
+    :language: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Pages/Default.html
+
+
+With the following partials:
+
+..  literalinclude:: _includes/_PartialBreadCrumb.html
+    :language: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Partials/Navigation/Breadcrumb.html
+
+And
+
+..  literalinclude:: _includes/_PartialMainNavigation.html
+    :language: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Partials/Navigation/MainNavigation.html
+
+
+..  _cobj-pageview-paths:
+
+paths
+-----
+
+..  confval:: paths.[priority]
+    :name: pageview-paths
+    :type: :ref:`data-type-path` with :ref:`stdWrap <stdwrap>`
+
+    Sets an array of paths for the Fluid templates, usually
+    :file:`EXT:my_extension/Resources/Private/Templates/` or a folder below that
+    path like :file:`EXT:my_extension/Resources/Private/Templates/MyPages`.
+
+    The templates are expected in a subfolder :path:`Pages` or
+    :path:`pages`.
+
+    Fluid partials are looked up in a sub-directory called :path:`Partials` or
+    :path:`partials`, layouts in :path:`Layouts` or :path:`layouts`.
+
+    The name of the used page layout (:ref:`Backend layout <t3coreapi:be-layout>`)
+    is resolved automatically.
+
+    The paths are evaluated from highest to lowest priority.
+
+
+..  _cobj-pageview-paths-example:
+
+Example: Define a path that contains all templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a basic definition of a :fluid:`PAGEVIEW` object with only one path to the
+templates:
+
+..  code-block:: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    page = PAGE
+    page.10 = PAGEVIEW
+    page.10.paths.100 = EXT:my_sitepackage/Resources/Private/Templates/
+
+The content of the folder could look like this:
+
+..  directory-tree::
+    :level: 2
+
+    *   EXT:my_sitepackage/Resources/Private/Templates/
+
+        *   Layouts
+
+            *   Default.html
+            *   WithoutHeader.html
+
+        *   Pages
+
+            *   Default.html
```

Comment:
Here we can add a hint, that the first letter has to be upper case. At least for now...

Reply:
I added a note